CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 Based on Realm JS v10.21.0: See changelog below for details on enhancements and fixes introduced between this and the previous pre release (which was based on Realm JS v10.19.5).
 
 ### Breaking change
+* Replaced `realm.empty` with `realm.isEmpty`
```

Comment:
```suggestion
* Replaced `Realm.empty` with `Realm.isEmpty`
```

Reply:
```suggestion
* Replaced `Realm#empty` with `Realm#isEmpty`
```

Whenever you want to talk about an instance property (like `empty` is a property of an instance of `Realm`) you use the `#` symbol avoid confusion with its static methods.

I've tried to, but failed at finding a reference for this. As far as I remember, I saw this pattern arising with the Ruby on Rails community.